sharding/networksharding/peerShardMapper.go

```diff
@@ -283,6 +295,32 @@ func (psm *PeerShardMapper) UpdatePeerIdShardId(pid core.PeerID, shardId uint32)
 	psm.fallbackPidShard.HasOrAdd([]byte(pid), shardId, uint32Size)
 }
 
+// UpdatePublicKeyPIDSignature updates the pid-signature pair associated with a public key
+func (psm *PeerShardMapper) UpdatePublicKeyPIDSignature(pk []byte, pid []byte, signature []byte) {
```

Comment:
Might change  pid []byte to pid core.PeerID in order to avoid confusions between parameters

Reply:
Done.